### DISCUSSION TITLE
ait-wiki/obtainingtardis/

### DISCUSSION THREAD

**Discussion by giscus** [amblelabs/ait-wiki#3] [General]
# ait-wiki/obtainingtardis/

Growing A TARDIS Coral Some prior information, you will need to obtain soul sand and a nether star, we suggest you read this rift chunk webpage as it is necessary to find one of these in the world before you can begin the process. To grow a TARDIS, follow these step-by-step instructions:
To start, you will need to craft a TARDIS Coral (top right).
Once a rift chunk is located, you will need to place down a soul sand block that is exposed to direct skylight (middle right).

https://theo.is-a.dev/ait-wiki/obtainingtardis/

**Comment by CrayzKato** [comment]
what does "water space" mean?

**Reply by MaketendoDev** [reply]
its just means make a 3 by 3 water area around

**Reply by tardisfan999** [reply]
are there any visuals for it?

**Reply by tardisfan999** [reply]
nvm
i found it

**Comment by ToxiCookies** [comment]
Is there anywhere I can find a list of the protocols descriptions?

**Reply by ToxiCookies** [reply]
nevermind, found the linking items sections!

**Reply by MaketendoDev** [reply]
awsome glad you found out!

**Comment by jmoso345** [comment]
I for the life of me can not get the Engine Core to work

**Reply by jmoso345** [reply]
never mine I think I got it

**Reply by MaketendoDev** [reply]
Awesome

**Comment by cyborgblue11** [comment]
how do i get me tardis to work after i grow it???

**Reply by cyborgblue11** [reply]
i am comfused

**Reply by Rasttny** [reply]
make 3x3x3 square of water, place engine core in centre of the water square, flip power lever on console

**Reply by cyborgblue11** [reply]
and it hast to be in the tardis?

**Reply by cyborgblue11** [reply]
![Screenshot 2024-09-02 221725](https://github.com/user-attachments/assets/99cd472e-8ddc-4888-98a8-1834d02bb5d4)


like this?

**Reply by Rasttny** [reply]
a big hint is if a spinny star thing appears below in the void

**Reply by cyborgblue11** [reply]
ohhhh thank you
